.github/workflows/yocto-build-deploy.yml

```diff
@@ -983,9 +957,9 @@ jobs:
 
     env:
       # Variables provided via the selected GitHub environment
-      BALENACLOUD_API_URL: ${{ vars.BALENA_URL || matrix.environment }}
+      BALENACLOUD_API_URL: ${{ matrix.environment }}
       BALENACLOUD_SSH_PORT: ${{ vars.BALENACLOUD_SSH_PORT || '222' }}
-      BALENACLOUD_SSH_URL: ssh.devices.${{ vars.BALENA_URL || matrix.environment }}
+      BALENACLOUD_SSH_URL: ssh.devices.${{ matrix.environment }}
```

Comment:
So, just to be sure, we are not going with creating a balenaMachine environment in GitHub now for only testing to help us ease up on the mass configuration and instead people can explicitly provide an environment now in the `test_matrix`

Correct?

Reply:
So `matrix.environment == URL == GitHub Environment name`

Reply:
Sounds good.